docs/generated/oc_by_example_content.adoc

```diff
@@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
 ====
 
 
+== oc secrets new-sourcecfg
+Create a new source secret
```

Comment:
I'm not opposed to there being a helper command that can generate based on
the inputs.  I'm not sure it should be called new-sourcecfg in that case -
it's more of a general secret generator, in which case it might just be
new.  However, we separate them today so you get nice help and
explanations.  Being able to add a file to an existing secret is generally
useful, so I'd rather support that as an option to a new-userpass (i don't
like that name) or new-sshkey (i don't like that name either).

On Wed, Sep 9, 2015 at 10:42 AM, Cesar Wong <notifications@github.com>
wrote:

> In docs/generated/oc_by_example_content.adoc
> <https://github.com/openshift/origin/pull/4433#discussion_r39050935>:
>
> > @@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
> >  ====
> >
> >
> > +== oc secrets new-sourcecfg
> > +Create a new source secret
>
> yup, unless you have your own server verification, I don't think that git
> will use a cacert to verify a ssh server
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4433/files#r39050935>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift